source/jobs/prerequisites/oracle.txt

```diff
@@ -10,13 +10,13 @@ Configure Migration Prerequisites for Oracle
    :depth: 1
    :class: singlecol
 
-To run migration jobs from an Oracle source database, the database may require 
-some configuration changes. If Relational Migrator determines the 
+When migrating data from an Oracle source database, the database may require 
+configuration changes. If Relational Migrator determines the 
```

Comment:
Perhaps, 

"Relational Migrator automatically checks your database for needed configuration changes and generates a SQL script to implement them."